File: app/src/main/java/com/droidtitan/volleyexamples/rest/util/Bus.java
package com.droidtitan.volleyexamples.rest.util;

import de.greenrobot.event.EventBus;

public final class Bus {
    private Bus() {
    }

    private static final EventBus DEFAULT_BUS = EventBus.getDefault();

    public static void register(Object target) {
        DEFAULT_BUS.register(target);
    }

    public static void unregister(Object target) {
        DEFAULT_BUS.unregister(target);
    }

    public static void post(Object event) {
        DEFAULT_BUS.post(event);
    }

}

File: app/src/main/java/com/droidtitan/volley/util/Bus.kt
package com.droidtitan.volley.util

import de.greenrobot.event.EventBus

/** A singleton object of Bus, this is created using. "object" instead of class. */
object Bus {
  private val DEFAULT_BUS = EventBus.getDefault()

  fun register(target: Any): Unit = DEFAULT_BUS.register(target)

  fun unregister(target: Any): Unit = DEFAULT_BUS.unregister(target)

  fun post(event: Any): Unit = DEFAULT_BUS.post(event)
}
